### ISSUE TITLE
Installation fails on Mac

### ISSUE THREAD

**Issue by avmaint** [Elektordi/obs-websocket-py#17]
running Python 3.5.2.

$ pip install obs-websocket-py

Collecting obs-websocket-py
  Downloading obs-websocket-py-0.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/5b/r380sbgs64d9v36w3qdn27m40000gn/T/pip-build-2q0t90dj/obs-websocket-py/setup.py", line 9, in <module>
        from generate_classes import generate_classes
    ImportError: No module named 'generate_classes'

**Comment by WolfgangAxel** [comment]
Same issues with me on Linux. I got past that by adding a quick and dirty path append before the import:

    ...
    from sys import path
    from os.path import dirname
    path.append(dirname(__file__))
    
    from generate_classes import generate_classes
    ...

I'm sure there's a better/more elegant way of doing this; I was just trying to force my way through the install with `pip`.

There are still problems further down the line though. In `~/obswebsocket/core.py`, it tries to do `from . import events`, but there is no `~/obswebsocket/events.py` to import. Is there a file missing from this repo, or should `events` be imported from somewhere else?

I suppose that should probably be its own issue. If it's more than just a missing file from the repo, I'll open a new one.

**Comment by MikeJewski** [comment]
I took events.py from version 0.2, and got another error, seems something might be messed up. 0.2 still installs fine though.

**Comment by WolfgangAxel** [comment]
Looks like the `events.py` from version 0.2 works, it just needed some names updated. [I have successfully installed obs-websocket-py with pip using these changes](https://github.com/WolfgangAxel/obs-websocket-py), and have put in a pull request. For anyone wanting to try installing it, download and extract the repo (or clone it using `git`), then run `pip` pointing to the directory. For example,

    pip install ~/Downloads/obs-websocket-py/

**Comment by ethaniel** [comment]
Same problem on windows:

```
C:\Users\Ark\Downloads\obscontrol>pip install obs-websocket-py
Collecting obs-websocket-py
  Using cached https://files.pythonhosted.org/packages/2c/82/125306f049d7517f6cbae078fe0dc4cb82df08d5a2b1018b7bbd753eadc8/obs-websocket-py-0.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "c:\users\ark\appdata\local\temp\pip-build-wisnkj\obs-websocket-py\setup.py", line 9, in <module>
        from generate_classes import generate_classes
    ImportError: No module named generate_classes

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in c:\users\ark\appdata\local\temp\pip-build-wisnkj\obs-websocket-py\
You are using pip version 9.0.3, however version 10.0.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

```

**Comment by ErikBorra** [comment]
Same issue on raspbian, python 2.7:

```
pip install obs-websocket-py
Collecting obs-websocket-py
  Downloading https://files.pythonhosted.org/packages/2c/82/125306f049d7517f6cbae078fe0dc4cb82df08d5a2b1018b7bbd753eadc8/obs-websocket-py-0.3.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-f35XrB/obs-websocket-py/setup.py", line 9, in <module>
        from generate_classes import generate_classes
    ImportError: No module named generate_classes
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-f35XrB/obs-websocket-py/
```

**Comment by Elektordi** [comment]
Hello,
(Sorry for the delay... :/ )

Just released version 0.4, it fixes a bug in the setup process, could you try again?

Best regards,

**Comment by Elektordi** [comment]
No news is good news... Fell free to open it again if the problem is still here!
(I haven't got a Mac, so I cannot test myself...)
